CHANGES

```diff
@@ -89,6 +89,10 @@ Documentation
   moved to the GitHub wiki section of this repository; the main entry
   page is https://github.com/coq/coq/wiki/The-Coq-FAQ.
 
+Standard Library
+
+- Added [Coq.Strings.String.concat] to join a list of strings with a separator
```

Comment:
The final join is hidden in this sentence.

Reply:
Updated